packages/usbutils.rb

```diff
@@ -37,7 +37,7 @@ def self.install
 
   def self.postinstall
     FileUtils.mkdir_p "#{CREW_PREFIX}/share/hwdata/" unless Dir.exist? "#{CREW_PREFIX}/share/hwdata/"
-    system "curl -#LO http://www.linux-usb.org/usb.ids -O #{CREW_PREFIX}/share/hwdata/usb.ids"
+    system "curl -#LO http://www.linux-usb.org/usb.ids -o #{CREW_PREFIX}/share/hwdata/usb.ids"
```

Comment:
Should we have a sha256 checksum on this?

Reply:
Actually, never mind.  It looks like this file is updated frequently.

Reply:
I'm wondering if this should be part of the install in addition to the postinstall to track the file for removal.

Reply:
You could probably do something as simple as:
```
FileUtils.touch "#{CREW_DEST_PREFIX}/share/hwdata/usb.ids"
```